services/storage/tests/_test_rawdatcore.py

```diff
@@ -18,7 +18,21 @@
 
 if utils.has_datcore_tokens():
     client = DatcoreClient(api_token=api_token, api_secret=api_secret)
+    api_secret = os.environ.get("BF_API_SECRET", "none")
+    destination = str(Path("MaG/level1/level2"))
+    fd, path = tempfile.mkstemp()
+
+    try:
+        with os.fdopen(fd, 'w') as tmp:
+            # do stuff with temp file
+            tmp.write('stuff')
+
+        f = client.upload_file(destination, path)
```

Comment:
Why don't you do your interface ``Path``-friendly instead of converting constantly to ``str``.